ziti/edge/network/factory.go

```diff
@@ -76,7 +76,7 @@ func (conn *routerConn) BindChannel(binding channel.Binding) error {
 	return nil
 }
 
-func (conn *routerConn) NewConn(service *rest_model.ServiceDetail, connType ConnType) *edgeConn {
+func (conn *routerConn) NewDialConn(service *rest_model.ServiceDetail, connType ConnType) *edgeConn {
```

Comment:
seems that connType is redundant here

Reply:
I think it used to be one method, and I didn't notice this when I split it into two. Fixed.